wwwbase/ajax/scramble.php

```diff
@@ -0,0 +1,201 @@
+<?php
+require_once("../../phplib/util.php");
+setlocale(LC_ALL, "ro_RO.utf8");
+ini_set('memory_limit','-1');
+
+
+//$searchWord = util_getRequestParameter('searchWord');
+//$option = $_GET['difficulty'];
+//var_dump($option);
+
+define("NIVEL_FOARTE_USOR", 4);
+define("NIVEL_USOR", 5);
+define("NIVEL_NORMAL", 6);
+define("NIVEL_GREU", 7);
+define("NIVEL_DICTIONAR", 7);
```

Comment:
Folosesc asa pentru ca in functie de nivelul de dificultate venit prin ajax, selectez un cuvant de o anumita lungime in functie de nivelul de dificultate ales. Deci daca vine 1 am nevoie de un cuvant de lungime 4. 
Am vrut sa fie mai evident ce reprezinta si cum sunt corelate acele numere 4,5,6,7 cu dificultatea.